Library/Homebrew/resource.rb

```diff
@@ -93,6 +92,7 @@ def apply_patches
   # A target or a block must be given, but not both.
   def unpack(target = nil)
     mktemp(download_name) do |staging|
+      fetch
```

Comment:
This is only used in `Patch#apply`. How about calling `fetch` before `resource.unpack` there?